source/fe/mapping_q_generic.cc

```diff
@@ -353,8 +353,9 @@ namespace internal
 
         // Construct the tensor product polynomials used as shape functions for the
         // Qp mapping of cells at the boundary.
+        QGaussLobatto<1> line_support_points(data.polynomial_degree+1);
         const TensorProductPolynomials<dim>
-        tensor_pols (Polynomials::generate_complete_Lagrange_basis(data.line_support_points.get_points()));
+        tensor_pols (Polynomials::generate_complete_Lagrange_basis(line_support_points.get_points()));
```

Comment:
Is this function only called once at the beginning of the mapping? If so, this is fine, but if it is called every time we map something, then setting up the `QGaussLobatto` every time may be expensive.